src/docs/userRegisterDocs.ts

```diff
@@ -170,3 +170,77 @@
  *         description: An error occurred while deleting the record.
  */
 
+
+/**
+ * @swagger
+ * /api/v1/updateProfile/{id}:
```

Comment:
since it coming from userRoutes you have to write this endpoint like this `/api/v1/user/updateProfile/{id}`

Reply:
yes, finished